cmd/list_schedules.go

```diff
@@ -4,22 +4,30 @@ import (
 	"fmt"
 
 	"github.com/form3tech-oss/go-pagerduty-oncall-report/api"
+
 	"github.com/spf13/cobra"
 )
 
 var listSchedulesCmd = &cobra.Command{
 	Use:   "schedules",
 	Short: "list schedules on PagerDuty",
 	Long:  "Get the list of schedules configured in PagerDuty",
-	RunE:  listSchedules,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		pd := &pagerDutyClient{client: api.NewPagerDutyAPIClient(Config.PdAuthToken)}
+		err := pd.listSchedules()
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
```suggestion
		return pd.listSchedules()
```

isn't it easier to return err right away?

Reply:
fixed